### PR TITLE
[NeuralChat] Specify local model path for nightly UT

### PR DESCRIPTION
**Pull request by letonghan**
## Type of Change

modify UT model path

## Description

Specify local model path for nightly UT

## Expected Behavior & Potential Risk

Nightly UT will load model from local path without downloading models. This will save disk space on testign servers.

## How has this PR been tested?

Local tested.

## Dependency Change?

no.

### CODE REVIEW

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/NeuralChat/job/nlp-toolkit-neuralchat-ut/112/

**Comment by letonghan** [comment]
rebuild after refactor: https://inteltf-jenk.sh.intel.com/view/NeuralChat/job/nlp-toolkit-neuralchat-ut/113/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/NeuralChat/job/nlp-toolkit-neuralchat-ut/115/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-neuralchat-ut/116/

**Review by lvliang-intel — APPROVED** [review]

File: intel_extension_for_transformers/neural_chat/requirements.txt [hunk 1]
```diff
@@ -70,3 +70,4 @@ zhconv
 urllib3
 langid
 diffusers==0.12.1
+transformers_stream_generator
```
[the remark this answers is not in the document]
Comment (letonghan) [reply intel_extension_for_transformers/neural_chat/requirements.txt:73]: It's for UT in test_qwen.py.

**Pull request merged**